commit 87e61ad56e79fccfaeac3ad7e04ee1a179d5a005
Author: Sultan Alsawaf <sultan@kerneltoast.com>
Date:   Tue Dec 29 15:09:47 2020 +0200

    fs: Align file struct to 8 bytes
    
    This improves hackbench results by about 5% on average.
    
    Signed-off-by: Sultan Alsawaf <sultan@kerneltoast.com>
    Signed-off-by: Forenche <prahul2003@gmail.com>

diff --git a/include/linux/fs.h b/include/linux/fs.h
index afbe4c266b44..03acc4c08c88 100644
--- a/include/linux/fs.h
+++ b/include/linux/fs.h
@@ -1034,7 +1034,7 @@ struct file {
 	ANDROID_KABI_RESERVE(1);
 	ANDROID_KABI_RESERVE(2);
 } __randomize_layout
-  __attribute__((aligned(4)));	/* lest something weird decides that 2 is OK */
+  __attribute__((aligned(8)));
 
 struct file_handle {
 	__u32 handle_bytes;
